### PR TITLE
bugfix: the VMethod push should returns empty string.

### PR DESCRIPTION
**Pull request by doujiang24**

### CODE REVIEW

**Comment by doujiang24** [comment]
travis failed when run test with perl 5.8: `Devel::Cover 0.001 is not supported on perl 5.008009`.
https://travis-ci.org/openresty/lemplate/jobs/223442906
so I removed perl 5.8, hope this is acceptable :)

**Comment by agentzh** [comment]
@doujiang24 We need to test perl 5.8. We just need to remove the `Devel::Cover` thing which was inherited from Jemplate. We don't really need it.

I've already fixed the travis CI build of this project in master.

File: t/vmethods.t [hunk 1]
```diff
@@ -39,7 +39,8 @@ Jack-Jill
 
 === TEST 4: push
 --- tt2
-[% array = [ 'Jack', 'Jill' ]; array.push('Jump').join('-') %]
+[% array = [ 'Jack', 'Jill' ]; array.push('Jump') %]
+[% array.join('-') %]
 --- out chomp
```
Comment (agentzh) [inline t/vmethods.t:44]: Will you add some tests for pushing multiple elements into the array?

File: .travis.yml [hunk 2]
```diff
@@ -5,7 +5,6 @@ perl:
 - 5.14
 - 5.12
 - 5.10
-- 5.8
```
Comment (agentzh) [inline .travis.yml]: We no longer need this since the master is already working with perl 5.8 in traivs ci.

**Review by agentzh — CHANGES_REQUESTED** [review]

**Comment by doujiang24** [comment]
@agentzh All done & rebased :)

**Comment by agentzh** [comment]
@doujiang24 Merged. Thanks!

**Pull request closed without merging**